<ui version="4.0" >
 <class>PrefMoveList</class>
 <widget class="QWidget" name="PrefMoveList" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupAnimations" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="title" >
      <string>&amp;Animations</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="5" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Rough
(little CPU usage)</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy" >
           <cstring>sliderSmoothness</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderSmoothness" >
          <property name="maximum" >
           <number>32</number>
          </property>
          <property name="pageStep" >
           <number>4</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Smooth
(high CPU usage)</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy" >
           <cstring>sliderSmoothness</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Animation s&amp;peed:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy" >
         <cstring>sliderSpeed</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Slow</string>
          </property>
          <property name="buddy" >
           <cstring>sliderSpeed</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderSpeed" >
          <property name="maximum" >
           <number>32</number>
          </property>
          <property name="pageStep" >
           <number>4</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Fast</string>
          </property>
          <property name="buddy" >
           <cstring>sliderSpeed</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="checkMovements" >
        <property name="text" >
         <string>&amp;Movements</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="checkFading" >
        <property name="text" >
         <string>&amp;Fading</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="checkHighlighting" >
        <property name="text" >
         <string>&amp;Smooth highlighting</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Animation s&amp;moothness:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy" >
         <cstring>sliderSmoothness</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>&amp;Colors and fonts</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>C&amp;omments color:</string>
        </property>
        <property name="buddy" >
         <cstring>colorComments</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="checkCommentsFont" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Cus&amp;tom comments font:</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="checkMovesFont" >
        <property name="text" >
         <string>C&amp;ustom moves font:</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>S&amp;elected move color:</string>
        </property>
        <property name="buddy" >
         <cstring>colorSelected</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KColorButton" name="colorSelected" />
      </item>
      <item row="2" column="1" >
       <widget class="KFontRequester" native="1" name="fontMoves" >
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KColorButton" name="colorComments" />
      </item>
      <item row="3" column="1" >
       <widget class="KFontRequester" native="1" name="fontComments" >
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>groupAnimations</tabstop>
  <tabstop>checkMovements</tabstop>
  <tabstop>checkFading</tabstop>
  <tabstop>checkHighlighting</tabstop>
  <tabstop>sliderSpeed</tabstop>
  <tabstop>sliderSmoothness</tabstop>
  <tabstop>colorSelected</tabstop>
  <tabstop>colorComments</tabstop>
  <tabstop>checkMovesFont</tabstop>
  <tabstop>fontMoves</tabstop>
  <tabstop>checkCommentsFont</tabstop>
  <tabstop>fontComments</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkMovesFont</sender>
   <signal>toggled(bool)</signal>
   <receiver>fontMoves</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>269</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel" >
     <x>359</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCommentsFont</sender>
   <signal>toggled(bool)</signal>
   <receiver>fontComments</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>282</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel" >
     <x>370</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
